README.md

```diff
@@ -71,13 +79,15 @@ python -m swarm_rl.train --env=quadrotor_multi --train_for_env_steps=1000000000
 --replay_buffer_sample_prob=0.75 --anneal_collision_steps=300000000 --experiment=swarm_rl 
 ```
 
-Or, even better, you can use the runner scripts in `swarm_rl/runs/`. Runner scripts (a Sample Factory feature) are Python files that
+We also provide a training script `train.sh`, so you can simply start training by command `bash train.sh`.
```

Comment:
Did you include train.sh in this PR? If not, please collaborate with Zhaojing to include train.sh into this PR.